ycsb-rs/src/main.rs

```diff
@@ -141,10 +141,7 @@ fn main() -> Result<()> {
         println!("[OVERALL], Throughput(ops/sec), {throughput}\n");
     }
 
-    monitor::print_client_matrics();
-    // monitor::print_proxy_matrics();
-    monitor::print_rpc_matrics();
-    monitor::print_rpc_num_matrics();
+    OBKV_CLIENT_REGISTRY.lock().unwrap().print_registry();
```

Comment:
Shall we provide a method in the client library to output all the metrics? Anyone can collect the metrics of the client if so.